plasma/root_chain/contracts/RootChain/RootChain.sol

```diff
@@ -245,11 +224,30 @@ contract RootChain {
         return (childChain[blockNumber].root, childChain[blockNumber].created_at);
     }
 
-    function getExit(uint256 priority)
+    function getDepositBlock()
+        public
+        view
+        returns (uint256)
+    {
+        return currentChildBlock.sub(childBlockInterval).add(currentDepositBlock);
+    }
+
+    function getExit(uint256 utxoPos)
+        public
+        view
+        returns (address, uint256)
+    {
+        return (exits[utxoPos].owner, exits[utxoPos].amount);
+    }
+
+    function getNextExit()
         public
         view
-        returns (address, uint256, uint256)
+        returns (uint256, uint256)
     {
-        return (exits[priority].owner, exits[priority].amount, exits[priority].utxoPos);
+        uint256 priority = exitsQueue.getMin();
+        uint256 utxoPos = uint256(uint128(priority));
+        uint256 created_at = (priority - utxoPos) >> 128;
```

Comment:
We can just do `priority >> 128` since it'll shift off the least significant 128 bits anyway.